includes/admin/class-wc-admin.php

```diff
@@ -152,28 +143,6 @@ public function admin_redirects() {
 			exit;
 		}
 
-		// Setup wizard redirect.
```

Comment:
I suggest we update the docblock of this method to remove the part about the setup wizard to avoid any potential confusions in the future.

Reply:
@rodrigoprimo I think this came from a merge conflict with `master`.

Reply:
Sorry, maybe I wasn't clear in my previous comment. What I meant is that now that we removed everything related to the setup wizard from `WC_Admin::admin_redirects()`, I believe we should update the method docblock to reflect this change and remove all the references to the setup wizard. There is also a inline comment in the method that mentions the OBW that might need to be updated as well.

https://github.com/woocommerce/woocommerce/blob/8fe3bcb90edc05e3d44c3c4c9e84b1af346cb044/includes/admin/class-wc-admin.php#L121

Reply:
Removed the reference to the setup wizard.